source/ranges.tex

```diff
@@ -5866,7 +5866,7 @@
 \end{codeblock}
 
 \pnum
-The \grammarterm{typedef-name} \tcode{iterator_category} denotes:
+When present, the \grammarterm{typedef-name} \tcode{iterator_category} denotes:
```

Comment:
This is certainly an improvement, but I'd also like the "present only if ..." comment in the synopsis to be stated in the normative prose. It's easy to miss the comment in the synopsis.

Contrast with [thread.lock.scoped] where [LWG 2731](https://wg21.link/lwg2731) added an explicit "there is no member `mutex_type`" for the case where it's absent.

So something like:

```suggestion
If `Base` does not model `forward_range` there is no member `iterator_category`.
Otherwise, the \grammarterm{typedef-name} \tcode{iterator_category} denotes:
```

If you'd prefer to do that separately, I'm happy to approve this as-is.

Reply:
I think this suggestion is nicely explicit. Done.